esp8266-wordclock.ino

```diff
@@ -519,6 +547,97 @@ void strip_apply_mask(byte x[]) {
   }
 }
 
+void strip_test_sequence(int sequence_interval, int wait, uint32_t color) {
```

Comment:
deze kun je nu aanroepen in de `if (time_not_synced_yet) {`. Dan kan die flash leds wel weg